libbpf-tools/execsnoop.bpf.c

```diff
@@ -73,6 +73,9 @@ int tracepoint__syscalls__sys_enter_execve(struct trace_event_raw_sys_enter* ctx
 	event->args_size = 0;
 
 	ret = bpf_probe_read_user_str(event->args, ARGSIZE, (const char*)ctx->args[0]);
+	if (ret < 0) {
+		return 0
```

Comment:
Okay, I changed my mind. Let us try to code based on uapi specification.
Have you compiled the code before submit the patch? There is a missing ';' in the above return statement.

Reply:
Aha, sloppy, fixed, compile and self-test pass.